doc/metrics.md

```diff
@@ -0,0 +1,65 @@
+# Instance metrics
+LXD provides metrics for all running instances.
+These are useful for monitoring instances.
+The instance metrics are updated when calling the `/1.0/metrics` endpoint, and are cached for 15 seconds.
+Any call to `/1.0/metrics` within these 15 seconds will return the cached metrics in order to reduce the load.
+
+# Create metrics certificate
+The `/1.0/metrics` endpoint is a special one as it also accepts a `metrics` type certificate.
+This kind of certificate is meant for metrics only, and won't work for manipulation instances for example
+
+Here's how to create a new certificate (this is not specific to metrics):
+
+```bash
+openssl req -x509 -newkey rsa:2048 -keyout ~/.config/lxc/metrics.key -nodes -out ~/.config/lxc/metrics.crt -subj "/CN=lxd.local"
+```
+
+Now, this certificate needs to be added to the list of trusted clients:
+
+```bash
+lxc config trust add ~/.config/lxc/metrics.crt --type=metrics
+```
+
+# Add target to prometheus
+In order for prometheus to scrape from LXD, it has to be added to the targets.
+
+First, one needs to make sure that LXD has set `core.https_address`.
```

Comment:
First, one needs to ensure that `core.https_address` is set so LXD can be reached over the network.